Project/js/SMProject.js

```diff
@@ -20,6 +20,40 @@ app.controller('Calculator', ['$scope', function ($scope) {
     //         $scope.parameters.arrivalRate = 60.0 / $scope.parameters.arrivalTime;
     //     }
     // }, true);
+
+    $scope.stores = [
+        {
+            'name' : 'Jhon Doe',
+            'prob_arrival' : 0.5,
```

Comment:
Seems like you forgot to add the normalizing feature to ensure all the probabilities sum 1.

Reply:
Everything else looks good.